### ISSUE TITLE
Build on Windows10: undefined reference to `osdialog_message'

### ISSUE THREAD

**Issue by antoniograzioli** [VCVRack/Rack#106]
I have tried building on Windows, followed instructions in Readme.md more than one.

git submodule update --init WORKS

cd dep
make   
cd .. WORKS

make (In rack folder gives these errors:

build/src/gui.cpp.o: In function `rack::guiInit()':
C:\users\anton\desktop\vcv\rack/src/gui.cpp:221: undefined reference to `osdialog_message'
build/src/app/RackScene.cpp.o: In function `rack::RackScene::step()':
C:\users\anton\desktop\vcv\rack/src/app/RackScene.cpp:57: undefined reference to `osdialog_message'
collect2.exe: error: ld returned 1 exit status
make: *** [compile.mk:39: Rack.exe] Error 1

**Comment by TronicLabs** [comment]
maybe?
```bash
# update external submodules referenced
$ cd Rack # enter into the Rack folder
$ git submodule update --init # update the submodule
```

**Comment by AndrewBelt** [comment]
I must have forgotten to update the submodule. Do a `git pull` in the osdialog repo for now.

**Comment by antoniograzioli** [comment]
yep that worked, thanks!

**Comment by AndrewBelt** [comment]
Fixed in 1a334a2213ae7fdabd0bc10a9d946763fd3b19ff
